ecs-cli/modules/cli/compose/entity/service/service_test.go

```diff
@@ -909,3 +886,22 @@ func getDescribeServiceTestResponse(existingService *ecs.Service) *ecs.DescribeS
 	}
 	return emptyDescribeServiceResponse
 }
+
+func getTestTaskDef(taskDefID string) (taskDefArn string, taskDefinition, registerTaskDefResponse ecs.TaskDefinition) {
+	taskDefArn = "arn/" + taskDefID
+	taskDefinition = ecs.TaskDefinition{
+		Family:               aws.String("family"),
+		ContainerDefinitions: []*ecs.ContainerDefinition{},
+		Volumes:              []*ecs.Volume{},
+	}
+	registerTaskDefResponse = taskDefinition
+	registerTaskDefResponse.TaskDefinitionArn = aws.String(taskDefArn)
+
+	return taskDefArn, taskDefinition, registerTaskDefResponse
```

Comment:
This works fine, though in the interest of science I believe you can just say `return` since the named return values here have already been set.

Reply:
:P i will note this for next time (for science)